translations/en-us.yml

```diff
@@ -1812,7 +1812,7 @@ settings:
             save_success: 'Email added'
             title: 'Connected emails'
             unconfirmed_emails: 'Unconfirmed emails'
-            update_fail: 'Unable to update email'
+            update_fail: 'This email has been either deactivated or marked as spam.<br><br>Please email contact@cos.io for further assistance.'
```

Comment:
Best to use the support email in config `config.support.supportEmail` instead hardcoding it here; also please enclose the email in `a` tags i.e `<a href="mailto:{supportEmail}">{supportEmail}</a>`

Reply:
Of course. Let me make those changes, thank you Fabrice.